### PR TITLE
Update conversations.txt

### PR DESCRIPTION
**Pull request by Sinanzee**
Few sentences changed for style and flow and wrote a new scene for the player for when they go to buy their first ship from the shipyard. Let me know any thoughts

### CODE REVIEW

File: data/conversations.txt [hunk 1]
```diff
@@ -10,31 +10,33 @@
...
-	`	The loan broker's office is equally intimidating. Large computer monitors fill the walls, with stock quotes and other information spilling down them almost too fast to read. You occasionally glimpse the names of places that you have only seen in movies: Aldebaran. Tarazed. Earth.`
-	`	The pen is heavy as lead but glides almost without friction as you print your initials on the bottom of page after page of ominous legal documents and sign several dozen statements, notices, and agreements. The banker, a balding middle-aged man in a suit that you suspect costs more than the spaceship you are about to purchase, flips the pages past you quickly, uttering a curt, "Sign here. Now here," as he points to each page. He moves swiftly, mechanically.`
+	`The bank's architecture is what you would have called "futuristic" back when you dreamed that the future would be less squalid than the present: story above story of curves and spires and balconies, all in gleaming metal. A doorman stands by each of the heavy glass doors. Despite wearing your very best outfit you still feel inadequate next to their glittering uniforms.`
```
Comment (Bladewood) [inline data/conversations.txt]: I'd put a comma after outfit.

File: data/conversations.txt [hunk 2]
```diff
@@ -10,31 +10,33 @@
...
+	`	The loan broker's office is equally intimidating. Impressive computer monitors fill the walls, with stock quotes and finance data spilling down them almost too fast to read. You occasionally glimpse the names of destinations that you have only seen in movies: Aldebaran. Tarazed. Earth.`
```
Comment (Bladewood) [inline data/conversations.txt]: "Impressive" doesn't really feel right to me.

"Destination" doesn't work here. The point of that line is that it's talking about other planets, other worlds, not trade routes.

File: data/conversations.txt [hunk 3]
```diff
@@ -10,31 +10,33 @@
...
+	`	The pen is heavy as lead but glides almost without friction as you print your initials on the bottom of page after page of ominous legal documents and sign several dozen statements, notices, and agreements. The banker, a balding middle-aged man in a suit that you suspect costs more than the spaceship you are about to purchase, nimbly flips through this dense river of legalese, uttering a curt, "Sign here. Now here," as he points to each page. He moves swiftly, mechanically.`
```
Comment (Bladewood) [inline data/conversations.txt]: Too informal for the game.

File: data/conversations.txt [hunk 4]
```diff
@@ -10,31 +10,33 @@
...
 	`	You sign your name one final time:`
 	name
-	`	The elevator is so well-tuned that you do not even realize it is moving until it has deposited you back in the lobby. But as you leave the bank, you are smiling. This crazy adventure suddenly feels real to you. You are going to do it. You are finally going to get off this planet.`
-	`	Compared to the bank, you feel much more at home in the shipyard, walking among the rusted out hulks and newer ships that gleam in the sunlight. You smell grease and dirt and rocket fuel; wonderful smells. There are three ship models within your price range. Which one you choose will determine your future...`
+	`	The elevator is so well-tuned that you do not even realize it is moving until it has deposited you back in the lobby. You bounce out of its metal chassis. A smile, ear to ear, carries you through the lobby, past the shining doormen, and out into the street. It is not until the end of the block that you finally jump and shout. This crazy adventure suddenly feels real to you. You are going to do it. You are finally going to get off this planet.`
```
Comment (Bladewood) [inline data/conversations.txt]: Too much emotion. Note that in Endless Sky dialogue, your reactions and choices are mostly supposed to be left to the imagination of the player; it's not normally something that the game "decides" for you.

File: data/conversations.txt [hunk 5]
```diff
@@ -10,31 +10,33 @@
...
+	`	Compared to the bank, you feel much more at home in the shipyard, walking among the rusted out hulks and newer ships that gleam in the sunlight. You hold hands with the smell of grease and dirt and rocket fuel as you drift towards the mechanic. When you ask her what she has in your price range she laughs. "I get one of you about every week." she explains, her face slick with sweat and engine grease. "Yeah alright let's take a look" With that, she leads you to a section of the shipyard with their more modest selections. As you walk, the ships surrounding you go from pristine, to adequate, to worn in, and the two of you finall end up in front of three seasoned vessels. She clasps her arm around your shoulder, chews, and spits. 
+
+"What'll it be, kid?"`
```
Comment (Bladewood) [inline data/conversations.txt]: Shouldn't be so many newlines in between. Either join this part with the previous line, or put it one line after. Remember to put backticks, also.

File: data/conversations.txt [hunk 6]
```diff
@@ -10,31 +10,33 @@
...
+	`	The elevator is so well-tuned that you do not even realize it is moving until it has deposited you back in the lobby. You bounce out of its metal chassis. A smile, ear to ear, carries you through the lobby, past the shining doormen, and out into the street. It is not until the end of the block that you finally jump and shout. This crazy adventure suddenly feels real to you. You are going to do it. You are finally going to get off this planet.`
+	`	Compared to the bank, you feel much more at home in the shipyard, walking among the rusted out hulks and newer ships that gleam in the sunlight. You hold hands with the smell of grease and dirt and rocket fuel as you drift towards the mechanic. When you ask her what she has in your price range she laughs. "I get one of you about every week." she explains, her face slick with sweat and engine grease. "Yeah alright let's take a look" With that, she leads you to a section of the shipyard with their more modest selections. As you walk, the ships surrounding you go from pristine, to adequate, to worn in, and the two of you finall end up in front of three seasoned vessels. She clasps her arm around your shoulder, chews, and spits.
```
Comment (Bladewood) [inline data/conversations.txt]: ~~Did you just assume that I'd buy my ship from a *mechanic*?~~
This paragraph isn't something I agree with, but regardless...

That period after `every week` should be a comma.

There should be commas after `Yeah` and `alright`, and a period after `take a look`. Or, put a period after `alright`, and capitalize `let's`.

`adequate` isn't usually used to describe visual appearances, only function.
There should be a dash in `worn in` to make it an adjective.

`finall` -> `finally`

File: data/conversations.txt [hunk 7]
```diff
@@ -10,31 +10,33 @@
...
+
+"What'll it be, kid?"`
 
 conversation "flight check: no energy!"
 	scene "scene/engine2"
-	`You've heard enough horror stories about the sort of accidents that happen when starship systems malfunction to be completely convinced of the necessity of a pre-flight safety check. As you listen to the complex symphony of hums, rattles, and clicks of "<ship>" warming up, you run through a mental checklist. Computers: check. Life support: check.`
-	`	It is all going smoothly until it comes time to fire up your systems for a test run... and you discover that your generator does not produce enough power to power them. You'll have to either install some batteries, or switch to a more powerful generator.`	
+	`You have heard more than enough horror stories about the kinds of accidents that can happen when your starship systems malfunction and demand a non-negotiable pre-flight safety check. As you listen to the complex symphony of hums, rattles, and clicks of "<ship>" warming up, you run through a mental checklist. Computers: check. Life support: check.`
```
Comment (Bladewood) [inline data/conversations.txt]: That second part doesn't really fit here.

File: data/conversations.txt [hunk 8]
```diff
@@ -10,31 +10,33 @@
...
+	`	Everything goes as expected until it comes time to fire up your systems for a test run...and with a metallic whimper you discover that your generator simply cannot power the entirety of your vessel. With a sigh, you take stock of your options. You will have to either install some new batteries, or switch to a more powerful generator.`
```
Comment (Bladewood) [inline data/conversations.txt]: There should be a space after the ellipsis. Only remove the space when it's an intentional part - and that means stuff like a jammed signal: `C...aptain, it's j...mming o...r s...gnal!`

Never heard `take stock of` before.

File: data/conversations.txt [hunk 9]
```diff
@@ -10,31 +10,33 @@
...
+	`You have heard more than enough horror stories about the kinds of accidents that can happen when your starship systems malfunction and demand a non-negotiable pre-flight safety check. As you listen to the complex symphony of hums, rattles, and clicks of "<ship>" warming up, you run through a mental checklist. Computers: check. Life support: check.`
+	`	As you near the end of your checklist, the air begins to grow uncomfortably warm. As you continue working sweat drips onto your flight controls and the air grows thick with the smell of...is something burning? It reeks of ozone and hot metal, and your cameras are showing what looks like a thick gaseous cloud billowing from a bottlenecked exhaust near the engines. As you watch this, you wipe your forehead with the back of your hand and it comes away drenched. If your systems are running this hot while still in the atmosphere, they will definitely overheat the moment you hit hard vacuum. You weigh your options. You are going to have to either install cooling systems, or trade out some components for lower-heat alternatives.`
```
Comment (Bladewood) [inline data/conversations.txt]: This, like the other flight check conversations you've written, doesn't fit with Endless Sky's tone.

Also, again, ellipsis.

File: data/conversations.txt [hunk 10]
```diff
@@ -10,31 +10,33 @@
...
 	`	You sign your name one final time:`
 	name
-	`	The elevator is so well-tuned that you do not even realize it is moving until it has deposited you back in the lobby. But as you leave the bank, you are smiling. This crazy adventure suddenly feels real to you. You are going to do it. You are finally going to get off this planet.`
-	`	Compared to the bank, you feel much more at home in the shipyard, walking among the rusted out hulks and newer ships that gleam in the sunlight. You smell grease and dirt and rocket fuel; wonderful smells. There are three ship models within your price range. Which one you choose will determine your future...`
+	`	The elevator is so well-tuned that you do not even realize it is moving until it has deposited you back in the lobby. You bounce out of its metal chassis. A smile, ear to ear, carries you through the lobby, past the shining doormen, and out into the street. It is not until the end of the block that you finally jump and shout. This crazy adventure suddenly feels real to you. You are going to do it. You are finally going to get off this planet.`
```
Comment (Bladewood) [inline data/conversations.txt]: Too much emotion. Note that in Endless Sky dialogue, your reactions and choices are mostly supposed to be left to the imagination of the player; it's not normally something that the game "decides" for you.

File: data/conversations.txt [hunk 11]
```diff
@@ -10,31 +10,33 @@
...
+	`	Compared to the bank, you feel much more at home in the shipyard, walking among the rusted out hulks and newer ships that gleam in the sunlight. You hold hands with the smell of grease and dirt and rocket fuel as you drift towards the mechanic. When you ask her what she has in your price range she laughs. "I get one of you about every week." she explains, her face slick with sweat and engine grease. "Yeah alright let's take a look" With that, she leads you to a section of the shipyard with their more modest selections. As you walk, the ships surrounding you go from pristine, to adequate, to worn in, and the two of you finall end up in front of three seasoned vessels. She clasps her arm around your shoulder, chews, and spits.
```
Comment (Bladewood) [inline data/conversations.txt]: ~~Did you just assume that I'd buy my ship from a *mechanic*?~~
This paragraph isn't something I agree with, but regardless...

That period after `every week` should be a comma.

There should be commas after `Yeah` and `alright`, and a period after `take a look`. Or, put a period after `alright`, and capitalize `let's`.

`adequate` isn't usually used to describe visual appearances, only function.
There should be a dash in `worn in` to make it an adjective.

`finall` -> `finally`

File: data/conversations.txt [hunk 12]
```diff
@@ -10,31 +10,33 @@
...
+	`	Compared to the bank, you feel much more at home in the shipyard, walking among the rusted out hulks and newer ships that gleam in the sunlight. You hold hands with the smell of grease and dirt and rocket fuel as you drift towards the mechanic. When you ask her what she has in your price range she laughs. "I get one of you about every week." she explains, her face slick with sweat and engine grease. "Yeah alright let's take a look" With that, she leads you to a section of the shipyard with their more modest selections. As you walk, the ships surrounding you go from pristine, to adequate, to worn in, and the two of you finall end up in front of three seasoned vessels. She clasps her arm around your shoulder, chews, and spits. 
+
+"What'll it be, kid?"`
```
Comment (Bladewood) [inline data/conversations.txt]: Shouldn't be so many newlines in between. Either join this part with the previous line, or put it one line after. Remember to put backticks, also.

File: data/conversations.txt [hunk 13]
```diff
@@ -10,31 +10,33 @@
...
 
 conversation "flight check: no energy!"
 	scene "scene/engine2"
-	`You've heard enough horror stories about the sort of accidents that happen when starship systems malfunction to be completely convinced of the necessity of a pre-flight safety check. As you listen to the complex symphony of hums, rattles, and clicks of "<ship>" warming up, you run through a mental checklist. Computers: check. Life support: check.`
-	`	It is all going smoothly until it comes time to fire up your systems for a test run... and you discover that your generator does not produce enough power to power them. You'll have to either install some batteries, or switch to a more powerful generator.`	
+	`You have heard more than enough horror stories about the kinds of accidents that can happen when your starship systems malfunction and demand a non-negotiable pre-flight safety check. As you listen to the complex symphony of hums, rattles, and clicks of "<ship>" warming up, you run through a mental checklist. Computers: check. Life support: check.`
```
Comment (Bladewood) [inline data/conversations.txt]: That last part doesn't really fit here.

File: data/conversations.txt [hunk 14]
```diff
@@ -10,31 +10,33 @@
...
+	`	Everything goes as expected until it comes time to fire up your systems for a test run...and with a metallic whimper you discover that your generator simply cannot power the entirety of your vessel. With a sigh, you take stock of your options. You will have to either install some new batteries, or switch to a more powerful generator.`
```
Comment (Bladewood) [inline data/conversations.txt]: There should be a space after the ellipsis. Only remove the space when it's an intentional part - and that means stuff like a jammed signal: `C...aptain, it's j...mming o...r s...gnal!`

Never heard `take stock of` before.

**Comment by Bladewood** [comment]
[Unrelated, but per @tehhowch's request (with a pretty convincing reason), I've started putting my comments in reviews instead of just individual comments.]

File: data/conversations.txt (hunk 9 above)
Comment (Sinanzee) [reply data/conversations.txt]: Okidoke, I'll revise and resubmit the pull request

File: data/conversations.txt (hunk 1 above)
Comment (Sinanzee) [reply data/conversations.txt]: yeah you're right

**Comment by Brick63** [comment]
You can make fixes to a already submitted pull

File: data/conversations.txt (hunk 2 above)
Comment (Sinanzee) [reply data/conversations.txt]: I really just wanted to switch it for the sake of signifying that the PC is beginning to think of these places as potential destinations. 'Places' seems too nondescript, I wanted to make some sort of indication as to the PC's mindset. Which as you say might not be something that I should be speaking for.

File: data/conversations.txt (hunk 10 above)
Comment (Sinanzee) [reply data/conversations.txt]: Yeah you're right

File: data/conversations.txt (hunk 3 above)
Comment (Sinanzee) [reply data/conversations.txt]: is it the inclusion of the word legalese?

File: data/conversations.txt (hunk 11 above)
Comment (Sinanzee) [reply data/conversations.txt]: I mean the original paragraph is incredibly barebones and does pretty little to cement the player in what should be a momentous decision. I agree that the scene I wrote itself was probably a bad choice, and would almost certainly need to be scrapped altogether but i just feel like the two or so sentences describing your character's flight to freedom in the original is wasted potential. I get that we shouldn't be speaking for the player character to allow for players to project themselves onto him, but the player character _is_ a character. He was born on little boston. He was dissatisfied with his surroundings enough to want to leave for the stars. He's of modest means and feels out of place in a loan broker's office. It isn't much, but there's already characterization here, why not capitalize on it?

File: data/conversations.txt (hunk 12 above)
Comment (Sinanzee) [reply data/conversations.txt]: agreed

File: data/conversations.txt (hunk 11 above)
Comment (Sinanzee) [reply data/conversations.txt]: for the record, I agree with most of what you say

File: data/conversations.txt (hunk 13 above)
Comment (Sinanzee) [reply data/conversations.txt]: could you specify what you mean by last part?

Comment (Bladewood) [reply data/conversations.txt]: `and demand a non-negotiable pre-flight safety check.`

It's only per societal standards to do that, it's not something quite something that the player himself would "demand". Yes, even at the risk of his life.

**Comment by Pointedstick** [comment]
If it ain't broke, don't fix it.

File: data/conversations.txt (hunk 3 above)
Comment (Bladewood) [reply data/conversations.txt]: `this dense river of legalese`

File: data/conversations.txt (hunk 13 above)
Comment (Sinanzee) [reply data/conversations.txt]: I think I addressed it. I think I was just unclear as to who was demanding what from whom.

**Comment by Sinanzee** [comment]
I think one of the reasons EV Nova was so immersive was because it could be very expressive at times. For the record: the only reason I felt the need to make this pull request was because I thought there were so many wasted opportunities to make the world feel believable and lived in. Maybe I was too expressive, and maybe I went in the wrong direction but I can't help but feel there's missed potential here. Two short sentences to describe an important decision like "There are three ship models within your price range. Which one you choose will determine your future...`" just felt awkward and game-y and not descriptive enough.

Also phrases like "malfunction to be completely convinced of the necessity of' just didn't sound good in my head maybe its subjective, but I really didn't like em.

If the resounding opinion is that I'm wrong, I'll accept it but I at least want to talk about it first.

**Comment by ghoulavenger** [comment]
I don't think it matters too much what type of language style is used, only that it is consistent. This consistency applies to primarily to characters (e.g. people need to act like they're the person in question) and established lore (contradictions are bad). But it can also apply to dialogue in general, because if people use different styles frequently then the game loses that very immersion you were looking for to begin with. My two cents, it's not a big enough deal to force people to adopt a new writing style.

**Comment by endless-sky** [comment]
My impression is that a lot of people actually thought the writing in Nova was overly florid and wordy. Essentially, the text in Endless Sky is interactive fiction, and one of the main ways that most IF differs from novel prose is an extreme economy and restraint in the writing.

Also, unless you're a *really* good writer it's hard to do wordy descriptions without looking amateurish (inserting way too many adjectives and adverbs, trying to use a different dialog tag every time instead of sticking to "says," etc.).

**Pull request closed without merging**